modules/javafx.graphics/src/main/native-glass/gtk/glass_window.cpp

```diff
@@ -1436,8 +1266,12 @@ void WindowContextTop::set_icon(GdkPixbuf* pixbuf) {
     gtk_window_set_icon(GTK_WINDOW(gtk_widget), pixbuf);
 }
 
-void WindowContextTop::restack(bool restack) {
-    gdk_window_restack(gdk_window, NULL, restack ? TRUE : FALSE);
+void WindowContextTop::to_front() {
```

Comment:
Do you use `raise` and `lower` here because the sibiling parameter is always `NULL` ?

Reply:
Yes. But also `gdk_window_raise` and `gdk_window_lower` seems to better match the `toFront()` and `toBack()` description.

See:
https://tronche.com/gui/x/xlib/window/XRestackWindows.html
https://tronche.com/gui/x/xlib/window/XRaiseWindow.html

gdk functions calls those Xlib functions internally.